roles/openshift_cfme/tasks/storage/create_pvs.yml

```diff
@@ -0,0 +1,65 @@
+---
+# Create the required PVs for the App and the DB
+- name: Ensure we save the local NFS server
+  set_fact:
+    l_openshift_cfme_nfs_server: "{{ groups['oo_nfs_to_config'].0 }}"
```

Comment:
I'd prefer it if the `groups['oo_nfs_to_config']` moved to a default variable( inside `defaults/main.yml`) so that we are not tying the inventory to the internals of a role.

Reply:
I'd like to avoid adding logic statements into the defaults. nfs server will be either the oo_nfs_to_config value, or it can be one specified by the user for an external server.

Reply:
The only issue is that I cannot run this role unless I have a `groups['oo_nfs_to_config']` in my inventory.  If this moves to the `defaults/main.yml` like so:
```
openshift_cfme_nfs_server: "{{ groups['oo_nfs_to_config'].0 }}"
```

I can override this parameter without requiring my inventory have `oo_nfs_to_config` in any groups.  This is very helpful when testing.